packages/trader/src/AppV2/Components/OnboardingGuide/guide-container.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import Joyride, { CallBackProps, STATUS, Step } from 'react-joyride';
+import { Localize } from '@deriv/translations';
+import GuideTooltip from './guide-tooltip';
+
+type TGuideContainerProps = {
+    should_run: boolean;
+    onFinishGuide: () => void;
+};
+
+const GuideContainer = ({ should_run, onFinishGuide }: TGuideContainerProps) => {
+    const [step_index, setStepIndex] = React.useState(0);
+    const steps = [
```

Comment:
is is possible to extracts step array outside the Component?

Reply:
Yes

Reply:
You want into a separate file or just above?

Reply:
honestly I would put it in a seperate file just to make the main component more clean